firebase-perf/e2e-app/src/main/res/navigation/mobile_navigation.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="utf-8"?> <!-- Copyright 2021 Google Inc. All Rights Reserved. -->
+<navigation xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:id="@+id/mobile_navigation"
+    app:startDestination="@+id/navigation_home">
+
+    <fragment
+        android:id="@+id/navigation_home"
```

Comment:
Can we name the fragments by which is a slow fragment and which is a fast fragment? Eg: can we name fragments as "Slow Fragment" and  "Fast fragment"? (this is incase if some fragments are slow and some of them are fast). If not, can we make fragments slow and fast?

Reply:
Done, made dashboard fragment into "FastFragment" and notification fragment into "SlowFragment"